src/main/kotlin/step3/domain/car/Cars.kt

```diff
@@ -0,0 +1,20 @@
+package step3.domain.car
+
+import step3.domain.car.status.Position
+import step3.util.Utils
+
+data class Cars(val car: List<Car>) {
```

Comment:
자동차 일급 컬렉션 정의 👍 
```suggestion
data class Cars(val cars: List<Car>) {
```
컬렉션 타입은 항상 복수형으로 정의하는것을 권장 드려요.

Reply:
네, 코멘트 남겨주신 대로 수정하였습니다! 😃 - https://github.com/next-step/kotlin-racingcar/commit/19f62e5c3341b6d0f40441a1c129087530f0533d